codegen/method.go

```diff
@@ -862,19 +865,36 @@ func (ms *MethodSpec) setTypeConverters(
 		return nil
 	}
 
-	respFields := respType.(*compile.StructSpec).Fields
-	downstreamRespFields := downstreamRespType.(*compile.StructSpec).Fields
-
 	respConverter := &TypeConverter{
 		LineBuilder: LineBuilder{},
 		Helper:      h,
 	}
 
-	err = respConverter.GenStructConverter(downstreamRespFields, respFields, respTransforms)
-	if err != nil {
-		return err
+	respConverter.append("func convert", pascalCase(ms.Name), "ClientResponse(in ", downstreamMethod.ResponseType, ") ", ms.ResponseType, "{")
```

Comment:
we dont have a lint rule yet for 80 characters but we try to stick to it, can add some linebreaks here for readability.